vertical-pod-autoscaler/recommender/model/container.go

```diff
@@ -126,6 +132,33 @@ func (container *ContainerState) addMemorySample(sample *ContainerUsageSample) b
 	return true
 }
 
+// RecordOOM adds info regarding OOM event in the model as an artifical memory sample.
+func (container *ContainerState) RecordOOM(timestamp time.Time, requestedMemory ResourceAmount) error {
+	resourceAmount := float64(requestedMemory)
+	// Discard old OOM
+	if timestamp.Before(container.WindowEnd.Add(-1 * MemoryAggregationInterval)) {
+		return fmt.Errorf("OOM event will be discarded - it is too old (%v)", timestamp)
+	}
+	// If OOM is in current or next window max requested mem with last known sample.
```

Comment:
nit: I had to read the code anyway to understand this comment :)

Reply:
Done.